### DISCUSSION TITLE
ECR scanning takes forever

### DISCUSSION THREAD

**Discussion by njahrckstr** [deepfence/ThreatMapper#910] [Q&A]
Hi everyone. We have implemented ThreatMapper in our AWS environment and it is mostly working great. However, the ECR scanning seems to be never ending. I initiated the scans on 20 February and they all still say "In Progress". The secrets scan completely successfully though. I was wondering if anyone else has encountered this issue and if there was a fix.
![ECR scan](https://user-images.githubusercontent.com/41128138/221023491-944d7315-ea0b-4536-82fc-23312664c171.jpg)

**Comment by shyam-dev** [comment]
Hello @njahrckstr - Adding this as an issue https://github.com/deepfence/ThreatMapper/issues/911
